README.md

```diff
@@ -1,6 +1,5 @@
 # Nashville
 SharpestMinds project examining Davidson County TN real estate data.
-First pulling data from https://public.enigma.com/datasets/davidson-county-tennessee-primary-assessments/606b9411-936f-429a-b2b1-7cbbb95dc44f
-and http://www.padctn.org/services/recent-sales/
+Scraping data from http://www.padctn.org/
```

Comment:
Great Change. Small ask and not a blocker, but add hyperlink to README for website.